src/amo/components/Header/styles.scss

```diff
@@ -1,7 +1,14 @@
 @import '~amo/css/styles';
 
-.Header {
+.Header-new-hero {
+  background: $color-ink-80;
```

Comment:
Can you file a follow-up issue to make this more future proof, per https://github.com/mozilla/addons-frontend/pull/8662/files#r329161686 ?

Reply:
Yep, #8700